src/jswrapper.h

```diff
@@ -58,19 +58,27 @@ typedef enum {
 // number of bits needed for each argument bit
 #define JSWAT_BITS GET_BIT_NUMBER(JSWAT_MASK+1)
 
+#ifndef ESP8266
+#define PACKED_JSW_SYM PACKED_FLAGS
+#else
+// On the esp8266 we put the JswSym* structures into flash and thus must make word-sized aligned
+// reads. Telling the compiler to pack the structs defeats that, so we have to take it out.
+#define PACKED_JSW_SYM
+#endif
+
 /// Structure for each symbol in the list of built-in symbols
 typedef struct {
   unsigned short strOffset;
-  void (*functionPtr)(void);
   unsigned short functionSpec; // JsnArgumentType
-} PACKED_FLAGS JswSymPtr;
+  void (*functionPtr)(void);
```

Comment:
The reordering here is to pack data better without the PACKED_FLAGS and to allow normal word access to functionPtr